qiita_core/util.py

```diff
@@ -93,3 +102,26 @@ def wrapper(*args, **kwargs):
         with TRN:
             return func(*args, **kwargs)
     return wrapper
+
+
+def get_qiita_version():
+    """Returns the Qiita version and Git sha if present
+
+    Returns
+    ------
+    tuple (version, sha)
+        The Qiita version and SHA. SHA can be an empty string.
+    """
+    # the actual repo is the abspath of the current file without
```

Comment:
Is this true even if you install it using pip install and is not located in the actual git repo directory?